py/soundswallower/__init__.py

```diff
@@ -21,11 +21,13 @@
 import wave
 from typing import Optional, Tuple
 
-from ._soundswallower import Config  # noqa: F401
-from ._soundswallower import Decoder  # noqa: F401
-from ._soundswallower import Endpointer  # noqa: F401
-from ._soundswallower import FsgModel  # noqa: F401
-from ._soundswallower import Vad  # noqa: F401
+from ._soundswallower import (  # noqa: F401
```

Comment:
Ah, looking at the history, we did need F401 at some point, just not anymore.

Reply:
https://docs.astral.sh/ruff/rules/unused-noqa/

Reply:
ah, I had forgotten all about `__all__` and gotten out of the habit of using it, thanks for the reminder!

Reply:
I tend to avoid `__all__` in general, just like I never use `import *`, but in a top-level situation like this, where you're importing stuff into `__init__.py` just so others can do `from soundswallower import foo`, putting those names into `__all__` as you have done is the right thing to do.